client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx

```diff
@@ -379,6 +379,10 @@ const InvestigatedPersonInfo = (props: Props) => {
                                 {
                                     isReturnSick && <ComplexityIcon tooltipText={formatDate(previousDiseaseStartDate)} />
                                 }
+                                <PatientInfoItem testId='contactFromAboardId' name='קיים מגע חו"ל' value={contactFromAboardId ? yes : noInfo} />
```

Comment:
maybe you can remove the testId, if I remember right I think that in case there is no contact from abroad we won't show this so you can cover this with condition like the others

Reply:
fixed